### PR TITLE
Update README for v3.0.0

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,7 +2,7 @@ Cosmos-SDK
 License: Apache2.0
 
                                  Apache License
-                           Version 2.0, January 2004
+                           Version 2.0, January 2020
```
Comment (shahankhatch) [inline LICENSE]: This year should not change, since the year is based on the official approval date by the ASF.

Comment (alessio) [inline LICENSE]: This text should be the same as https://www.apache.org/licenses/LICENSE-2.0

```suggestion
                           Version 2.0, January 2004
```

**Review by shahankhatch — APPROVED** [review]
lgtm

**Review by alessio — APPROVED** [review]
Changes to LICENSE should be reverted. Otherwise it looks good!

File: LICENSE (hunk 1 above)
Comment (jackzampolin) [reply LICENSE]: cool

**Pull request merged**